### ISSUE TITLE
[4.0.7] Can't find husky, skipping pre-push hook on Windows 10

### ISSUE THREAD

**Issue by royk** [typicode/husky#643]
Windows 10
Git version 2.18.0.windows.1
Node version: v10.16.0

Starting with 4.0.7, Husky hangs when pushing:

```
HUSKY_DEBUG=1 git push
husky:debug husky v4.0.7 (created at 1/14/2020, 10:40:39 AM)
husky:debug pre-push hook started
husky:debug Current working directory is /c/....<redacted>
husky:debug pre-push hook started

```
When I ctrl+C I get:
```
husky:debug npx --no-install husky-run exited with 127 exit code
Can't find Husky, skipping pre-push hook
You can reinstall it using 'npm install husky --save-dev' or delete this hook
```

Rolling back to 4.0.6 fixes this issue

**Comment by typicode** [comment]
Hi @royk,

Thanks for the report. Which terminal or GUI are you using to push?

**Comment by royk** [comment]
Git for windows (https://gitforwindows.org/)

**Comment by typicode** [comment]
I'm reproducing the issue. @casz do you also have it?

**Comment by casz** [comment]
I assume we have to add a test for which package manager is yarn and version is less than 
`2`

**Comment by casz** [comment]
I'll see what I can cook up 🤔 as I have the code in my reflog 😆

**Comment by casz** [comment]
@royk @typicode feel free to test it wtih 
```
npm install git://github.com/casz/husky.git#fix/npx-in-winpty-dist --save-dev
```

**Comment by royk** [comment]
@casz Solved it on my end ✌️

**Comment by typicode** [comment]
@royk could you try with `husky@4.0.9`?

**Comment by casz** [comment]
also fixed by 4.0.9

**Comment by typicode** [comment]
Thanks for the PRs 👍
